### ISSUE TITLE
Listing of completed tasks is too limited

### ISSUE THREAD

**Issue by Jefficus** [agateau/yokadi#63]
With -d, I can list completed tasks from today, yesterday, this week, this month, or any specific day, but not from an arbitrary period of time. I often need last week, last month, and sometimes need a specific week from six months ago.

For last week, I could almost use lastmonth and just find the week I need, but the list doesn't show date completed.

It would be fairly helpful if I could at least use the syntax of the --due filter and specify things like > 2019-11-13. And if I could specify two of those, that would solve things completely, allowing me to set a before and after date filter for any period of time.

**Comment by agateau** [comment]
This is definitely a good idea. Are you interested in giving it a try?

**Comment by Jefficus** [comment]
I'm currently overcommitted, but I may find some time to look into it this winter. The tricky part will be implementing it in a way that doesn't clash with other date selection commands and options, and ideally enhances all of them.

**Comment by agateau** [comment]
Indeed, and there is probably some consolidation work to do on the date syntaxes
